docs/interactions/Slash_Commands.md

```diff
@@ -25,7 +25,7 @@ In this documentation you'll find some notes about limits and caps on certain pa
 
 ## What is a Slash Command
 
-A **Slash Command** is a command that you register for your application. They're made up of a name, description, and a block of `options`, which you can think of like arguments to a function. The name and description help users find your command among many others, and the `options` validate user input as they fill out your command.
+**Slash Commands** (synonymous with application commands) are a type of interaction that you can register for your application. They're made up of a name, description, and a block of `options`, which you can think of like arguments to a function. The name and description help users find your command among many others, and the `options` validate user input as they fill out your command.
```

Comment:
this is misleading: slash commands aren't really "a type of interaction", they are their own structure, and they create an interaction when they are used

Reply:
Idk how to accurately describe it. It's kinda explained at the top of the article right? maybe it would make more sense to remove the sentence all together

Reply:
> Идк как это точно описать. Это вроде как объясняется в верхней части статьи, верно? возможно, было бы разумнее удалить предложение все вместе

Файл не найден.

Reply:
You can probably just remove the first sentence and have it start with 

> Slash commands are made up of a name...

Reply:
You can probably just remove the first sentence and have it start with

> Slash commands are made up of a name...